modules/apps/transfer/types/msgs_test.go

```diff
@@ -98,3 +103,36 @@ func TestMsgTransferGetSigners(t *testing.T) {
 
 	require.Equal(t, []sdk.AccAddress{addr}, res)
 }
+
+// TestMsgUpdateParamsValidation tests ValidateBasic for MsgUpdateParams
+func TestMsgUpdateParamsValidation(t *testing.T) {
+	testCases := []struct {
+		name    string
+		msg     *types.MsgUpdateParams
+		expPass bool
+	}{
+		{"valid authority with valid params", types.NewMsgUpdateParams(validAuthority, types.DefaultParams()), true},
+		{"valid authority address with valid params", types.NewMsgUpdateParams(validAddress, types.DefaultParams()), true},
```

Comment:
I guess these testcases are more or less the same. We could remove `validAuthority` and `validAddress` and just use `ibctesting.TestAccAddress`

Reply:
done